Update/2.25.0 (#38)
Start9Labs/filebrowser-wrapper: README.md

- [docker](https://docs.docker.com/get-docker)
- [docker-buildx](https://docs.docker.com/buildx/working-with-buildx/)
- [deno](https://deno.land/#installation)
- [make](https://www.gnu.org/software/make/)
- [start-sdk](https://github.com/Start9Labs/start-os/tree/sdk/backend)
- [yq](https://mikefarah.gitbook.io/yq)

## Cloning

Clone the File Browser Wrapper locally.

```
git clone git@github.com:Start9Labs/filebrowser-wrapper.git
cd filebrowser-wrapper
```

## Building

To build the **File Browser** service as a universal package, run the following command:

```
make
```

Alternatively the package can be built for individual architectures by specifying the architecture as follows:

```
make x86
```

or

```
make arm
```

## Installing (on StartOS)

Before installation, define `host: https://server-name.local` in your `~/.embassy/config.yaml` config file then run the following commands to determine successful install:

> :information_source: Change server-name.local to your Start9 server address

```
start-cli auth login
#Enter your StartOS password
make install
```

> **Tip:** You can also install the filebrowser.s9pk using **Sideload Service** under the **StartOS > SETTINGS** section.

Reviewer: ```suggestion
**Tip:** You can also install the filebrowser.s9pk by sideloading it under the **StartOS > System > Sideload a Service** section.
```
